File: 01/solving.py
#!/usr/bin/env python3

def isBissextile(number):
    if(number %4 == 0):
        if(number % 100 == 0 and number % 400 != 0):
            return False
        return True
    return False

def main():
    value = int(input("ANO: "))
    print(isBissextile(value))

if __name__ == '__main__':
    main()

File: 01/solving.c
#include <stdio.h>
#include <stdbool.h>

bool isBissextile(int year) {
    if(year %4 == 0){
        if((year %100 == 0 ) && (year %400 !=0)){
            return false;
        }
        return true;
    }
    return false;
}

int main() {
    int year;
    printf("ANO: \n");
    scanf("%d", &year);
    
    
    printf("%B\n", isBissextile(year));
    return 0;
}